COMETwebapp/ViewModels/Components/Viewer/PropertiesPanel/IPropertiesComponentViewModel.cs

```diff
@@ -26,32 +26,25 @@ namespace COMETwebapp.ViewModels.Components.Viewer.PropertiesPanel
 {
     using CDP4Common.EngineeringModelData;
 
-    using COMETwebapp.Utilities;
     using COMETwebapp.Model;
-    
-    using Microsoft.AspNetCore.Components;
-    
-    using COMETwebapp.IterationServices;
-    using COMETwebapp.Services.SessionManagement;
     using COMETwebapp.Services.Interoperability;
+    using COMETwebapp.Services.SessionManagement;
+    using COMETwebapp.Utilities;
+
+    using Microsoft.AspNetCore.Components;
 
     /// <summary>
-    /// Interface for the <see cref="PropertiesComponentViewModel"/>
+    /// Interface for the <see cref="PropertiesComponentViewModel" />

```

Comment:
please document the intent of the interface instead of pointing to the implementation.